tortoise/full_test.go

```diff
@@ -321,7 +321,7 @@ func TestFullCountVotes(t *testing.T) {
 	} {
 		tc := tc
 		t.Run(tc.desc, func(t *testing.T) {
-			logger := logtest.New(t)
+			logger := logtest.Zap(t)
```

Comment:
I think we can do completely away with our wrapper:

```suggestion
			logger := zaptest.New(t)
```

Reply:
logtest also check env variable